### ISSUE TITLE
Docker image creates empty volume on start

### ISSUE THREAD

**Issue by jeffrson** [verdaccio/verdaccio#2991]
**Your Environment**
  
 * **verdaccio version**: 5.5.2
 * **platform**: [npm, docker, helm, other] docker

**Describe the bug**

On each `docker run` (also via docker compose) a *new* volume for /verdaccio/storage is created. IMO, it should be left to the user, whether and how the volume should be created at all. Besides, the empty volume on start is rather useless.
For me, I'd like to use a bind mount from a certain host directory to /verdaccio. However, at the moment, I need to override the volume at /verdaccio/storage in order to prevent those <docker volume id> volumes and, most notably, reuse all uploaded packages.

**To Reproduce**

```
docker run --rm -d verdaccio/verdaccio:5.5.2
docker volume ls
```

**Expected behavior**

Doesn't create any unexpected docker volume.

**Comment by juanpicado** [comment]
Works for me
```
docker --version
Docker version 20.10.12, build e91ed57

➜  docker run --rm -d verdaccio/verdaccio:5.5.2
543dc................................................5474
➜  docker volume ls                            
DRIVER    VOLUME NAME
local     .708b2e00f.................0b324e8............
```

**Comment by jeffrson** [comment]
Yes, and with next `docker run...` it will create another volume a.s.o., always forgetting about the previous.

**Comment by juanpicado** [comment]
> IMO, it should be left to the user, whether and how the volume should be created at all. 
> For me, I'd like to use a bind mount from a certain host directory to /verdaccio

Ok I see your point,I don't see the big issue, but anyway, do you have any specific proposal that improves what exist already?

**Comment by jeffrson** [comment]
The issue is "simple": after restarting verdaccio, all locally published npm packages are gone. They went away with the old inaccessible volume. All those private package will need to be republished.

Suggestion would be to remove volume definition from Dockerfile, since anonymous volumes are not very useful these days anyway. Everybody would be free to define a volume as they like, be it for /verdaccio or /verdaccio/storage, /verdaccio/config, /verdaccio/plugins.

**Comment by juanpicado** [comment]
I understand but I don't agree, I think are very useful, volumes can be mapped if you want to persist you have volumes, docker-compose, etc etc, anyhow I guess is just matter of opinions, I respect your of course. 

This is not a bug, rather a suggestion so I'll move this thread to a disscusion and tag the Helm/Kubernetters users which have more experience than me in this matter and keep discussing there.
